theano/gpuarray/dnn.py

```diff
@@ -1393,6 +1393,9 @@ def local_abstractconv_cudnn(node):
     inp1 = node.inputs[0]
     inp2 = node.inputs[1]
 
+    if (node.op.filter_dilation != (1, 1)):
```

Comment:
Wait, an interesting side effect of how we do it in Lasagne is that the *gradient* of the dilated convolution becomes a forward pass. This is something worth considering in the optimizer as well. Maybe we should reactivate the meta-optimizer, but then we need symbolic stride inference to make it work correctly.